modules/ROOT/pages/design-modify-raml-specs-conform.adoc

```diff
@@ -705,7 +740,7 @@ In this next example, there are two properties in a single example. However, the
   b: 1
 ----
 
-If you add the name of the type, then the warning message no longer appears:
```

Comment:
"a" and "b" properties are parsed as example properties and not as two different examples. Amf will inline the fragment at the places in which it has been included.

Reply:
How is the editor able to parse this without issuing a warning, seeing as there is no name?
`#%RAML 1.0 NamedExample
  a: "b"
  b: 1`
Won't the editor return a warning?

Reply:
There will be a warning because the fragment doesn't have a name, and it will be inlined. What I meant is you cannot know if a and b will be parsed as separated examples, that depends on where is used.